src/mauve/utils.py

```diff
@@ -27,6 +27,7 @@ def get_model(model_name, tokenizer, device_id):
     if 'gpt2' in model_name:
         model = AutoModel.from_pretrained(model_name, pad_token_id=tokenizer.eos_token_id).to(device)
         model = model.eval()
+        model = model.double()
```

Comment:
We want to use a `float64` model only for the testing the batched implementation. It is not a good idea to always use `float64`. Could you delete this line?